### PR TITLE
Works+about myself

### PR DESCRIPTION
**Pull request by FabricioLatl**
Pull request to merge the branch in which the sections of about me and skills have been created

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -52,7 +52,271 @@ <h3>
         <li><a href='https://www.linkedin.com/in/fabriciolatl/'><img alt="linkedin" src='images/linkedin.svg' /></a></li> 
         <li><a href='https://twitter.com/Wafer_lt'><img alt="twitter" src='images/twitter.svg' /></a></li>
       </ul>
+
+      <section class="recent-works-container">
+        <div class="recent-works-title">
+            <h2>My recent Works</h2>
```
Comment (muneebulrehman) [inline index.html:59]: I would recommend using the same font size/weight as well as same dimension for height as in the Figma template for the technologies section. Also, consider making the button more like as in Figma template.
| Your design  | Figma design |
| ------------- | ------------- |
| image | image |

File: index.html [hunk 2]
```diff
@@ -52,7 +52,271 @@ <h3>
         <li><a href='https://www.linkedin.com/in/fabriciolatl/'><img alt="linkedin" src='images/linkedin.svg' /></a></li> 
         <li><a href='https://twitter.com/Wafer_lt'><img alt="twitter" src='images/twitter.svg' /></a></li>
       </ul>
+
+      <section class="recent-works-container">
```
Comment (muneebulrehman) [inline index.html:56]: Your work cards section also does not match the Figma template. Kindly fix it and then ask for a re-review.
| Your design  | Figma design |
| ------------- | ------------- |
| image | image |

File: index.html [hunk 3]
```diff
@@ -52,7 +52,271 @@ <h3>
...
+            <button type="submit" class="see-project-button-full">
+              See Project
+            </button>
+          </li>
+        </ul>
+      </section>
+
+
+
+      <section class="about-me-container">
```
Comment (muneebulrehman) [inline index.html:229]: The dividing line between the about section and languages section is missing in your case. Consider adding it.
| Your design  | Figma design |
| ------------- | ------------- |
| image | image |

File: index.html [hunk 4]
```diff
@@ -52,7 +52,271 @@ <h3>
...
+        </div>
+        <p> 
+          Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+  
+        <button type="submit" class="see-project-button">GET MY REUSME</button>
+
+        <div class="spacer">.</div>
+
+        <div class="about-me-list">
```
Comment (muneebulrehman) [inline index.html:246]: It looks to me, that you have aligned the text to the start of the container where as in the Figma template it is aligned in the center. Consider fixing it.
| Your design  | Figma design |
| ------------- | ------------- |
| image | image |

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @FabricioLatl , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors

## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by FabricioLatl** [comment]
Hi @muneebulrehman, this peoject is for the mobile version of the portfolio. I’ve been INTENSIVELY told that the project should be for mobile on my last pull request, given that i had done the same mistake and developed for desktop.

Please respond to me to clear up any doubts

Thank you

**Comment by muneebulrehman** [comment]
@FabricioLatl , i am sorry. I missed the headline saying that it was for mobile view. I will update my comments accordingly.

**Review by OmarRamoun — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi @FabricioLatl 👋

Your project is complete! ✔️ 🏁. This is the First Project 1️⃣ You get Approved ✔️! in microverse 🌐.

Finally, **you did it**! 🎉 👏. Your code was very professional 💯 and I can see you have a great chance to be a Great Software Engineer ⛑️.

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉. And Welcome to Microverse 👋.

> **Note:** You can now take a break even if short before you move on to the next project. Happy 😄 Coding 🖥️! and Have a nice day 🌞.

## Highlights 🌟

- ✔️ Descriptive commmit messsages
- ✔️ Descriptive PR description
- ✔️ No linter Errors
- The code is top-notch! 💯 and have a huge potential.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@omarramoun](https://github.com/omarramoun)
- This is My **Twitter**: [@omarramoun](https://twitter.com/omarramoun)
- This is My **Slack**: [Ramoun](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**